Since we remove <utmp{,x}.h> from musl for utmps support, we can't use the
included <lastlog.h>.

This implements just the lastlog stuff from musl <utmp.h>.

--- musl-1.1.19/include/lastlog.h	2018-02-22 12:39:19.000000000 -0600
+++ musl-1.1.19/include/lastlog.h	2018-06-15 20:04:54.428854473 -0500
@@ -1 +1,23 @@
-#include <utmp.h>
+#ifndef _LASTLOG_H
+#define _LASTLOG_H
+
+#include <sys/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define UT_HOSTSIZE 256
+#define UT_LINESIZE 32
+
+struct lastlog {
+	time_t ll_time;
+	char ll_line[UT_LINESIZE];
+	char ll_host[UT_HOSTSIZE];
+};
+
+#ifdef __cplusplus
+}
+#endif
+
+#endif
